scipy/_lib/highs

```diff
@@ -0,0 +1 @@
+Subproject commit 2acc46c42f1641e1bdb58ab9cf846560bf408bd0
```

Comment:
This should not be in this PR

Reply:
Sorry, no idea how this file got there, but it is removed now

Reply:
No worries 😃